test/e2e/cloudstack_test.go

```diff
@@ -17,6 +17,46 @@ import (
 	"github.com/aws/eks-anywhere/test/framework"
 )
 
+// ApiServerExtraArgs
+func TestCloudStackKubernetes129RedHat8ApiServerExtraArgsSimpleFlow(t *testing.T) {
+	test := framework.NewClusterE2ETest(
+		t,
+		framework.NewCloudStack(t, framework.WithCloudStackRedhat129()),
+		framework.WithClusterFiller(api.WithKubernetesVersion(v1alpha1.Kube129)),
+		framework.WithClusterFiller(api.WithControlPlaneAPIServerExtraArgs("service-account-issuer", "https://my-custom-issuer-url")),
```

Comment:
This is a fake issuer right? It doesn't cause any issues?

Reply:
Yep, there are no issues in configuring any value for the flags. There are no checks for that